src/pages/landing/ebooks/components/_header.tsx

```diff
@@ -1,12 +1,29 @@
 import React from 'react'
 import styled from 'styled-components'
-import PropTypes from 'prop-types'
 import GetEbook from './_get-ebook'
 import { Flex } from 'components/containers'
 import { Header, QueryImage, Text } from 'components/elements'
 import { localize } from 'components/localization'
 import device from 'themes/device.js'
 
+type HeaderImageProps = {
+    imgWidth: number
+    imgHeight: number
+}
+
+type HeaderSectionProps = {
+    authorDesc: string
+    authorName: string
+    bg?: string
+    bgMobile?: string
+    ebook_utm_code: string
+    imgHeight: number
+    imgWidth: number
+    introMain: string
+    introSub: string
+    mainHeaderImage: any
```

Comment:
mainHeaderImage should use ImageDataLike from 'gatsby-plugin-image'

Type HeaderSectionProps the same with HeroProps. Try to find a way to not to duplicate code and better export type from one file
It's actual for ImageTextProps and IntroductionProps too

Reply:
All done.